iceoryx_posh/test/integrationtests/test_mq_interface_startup_race.cpp

```diff
@@ -97,8 +97,9 @@ class CMqInterfaceStartupRace_test : public Test
         constexpr uint32_t DUMMY_SHM_OFFSET{73};
         constexpr uint32_t DUMMY_SEGMENT_ID{13};
         constexpr uint32_t INDEX_OF_TIMESTAMP{4};
+        constexpr uint32_t ENABLE_MONITORING{true};
         regAck << IpcMessageTypeToString(IpcMessageType::REG_ACK) << DUMMY_SHM_SIZE << DUMMY_SHM_OFFSET
-               << oldMsg.getElementAtIndex(INDEX_OF_TIMESTAMP) << DUMMY_SEGMENT_ID;
+               << oldMsg.getElementAtIndex(INDEX_OF_TIMESTAMP) << DUMMY_SEGMENT_ID << ENABLE_MONITORING;
```

Comment:
Small nitpick. On the caller side this is called `m_sendKeepalive` and therefore this is semantically more of a `SEND_KEEP_ALIVE{true}` but it's not that important.

Reply:
How about creating a `SEND_KEEP_ALIVE {isMonitored}` in `process_manager.cpp` and sending that instead of `isMonitored`. Our future selves might be thankful for this information.

Reply:
Please also update the copyright